### ISSUE TITLE
Investigate about archives candidates JSON metadata

### ISSUE THREAD

**Issue by leplatrem** [mozilla-services/buildhub#19]
Build info are published as JSON in the candidates builds folder and for nightlies, but only for en-US.

https://archive.mozilla.org/pub/firefox/candidates/51.0b6-candidates/build1/linux-x86_64/en-US/firefox-51.0b6.json

https://archive.mozilla.org/pub/firefox/nightly/2017/05/2017-05-01-10-01-39-mozilla-central/firefox-55.0a1.en-US.linux-i686.json

Investigate:

* what info we can reuse for other locales?
* how the [Socorro scrapper](https://github.com/mozilla/socorro/blob/9c3ba26/socorro/cron/jobs/ftpscraper.py) uses the list of candidates builds?
* is the list of builds fixed or has a chance to be updated?

See #18

**Comment by glasserc** [comment]
- I strongly suspect that the JSON metadata is shared for all locales. See https://github.com/mozilla/socorro/blob/9c3ba26/socorro/cron/jobs/ftpscraper.py#L144 and https://github.com/mozilla/socorro/blob/9c3ba26/socorro/cron/jobs/ftpscraper.py#L164-L167. I don't see anything in the Socorro ftpscraper that refers to locales explicitly at all, nor any "locale" fields in the table that Rémy posted in the blueprint.

- The ftpscraper uses the latest build, i.e. the last element in the "candidates" directory (for example https://archive.mozilla.org/pub/firefox/candidates/51.0b6-candidates). See https://github.com/mozilla/socorro/blob/9c3ba26/socorro/cron/jobs/ftpscraper.py#L133. I guess other builds are ignored. (I don't see anything about "builds" for nightlies, only releases.)

- It seems like the list of builds comes from whatever is on the server and ftpscraper just looks for anything starting with "build". See https://github.com/mozilla/socorro/blob/9c3ba26/socorro/cron/jobs/ftpscraper.py#L129. (Again, I don't see anything about builds in nightlies.)

**Comment by Natim** [comment]
Apparently the buildid is specific to the platform but not to the locale.
Aso you can see by looking at the platform_info.txt there: https://archive.mozilla.org/pub/firefox/candidates/54.0b9-candidates/build1/

The idea is to improve the scrapping script so that it would provision the buildid for all locales.

**Comment by mostlygeek** [comment]
> Apparently the buildid is specific to the platform but not to the locale.
Aso you can see by looking at the platform_info.txt there: https://archive.mozilla.org/pub/firefox/candidates/54.0b9-candidates/build1/

Should change the schema so `locales` is a a top level key in the record? 

For example:

```
source: ...
build: ...
target: ...
locales: {
   en-US: {
      download: [object]
   }
}
```

It is safe to assume that each locale has the same: 

- systemaddons
- target
- source

**Comment by leplatrem** [comment]
Fixed in #136
